ChangeLog

```diff
@@ -3,8 +3,8 @@ mbed TLS ChangeLog (Sorted per branch, date)
 = mbed TLS x.x.x branch released xxxx-xx-xx
 
 Bugfix
-   * return error code of `mbedtls_aes_crypt_ecb` if failed, when calling it from
-     other modes. FIx for #1092 reported by adustm
+   * Return error code of `mbedtls_aes_crypt_ecb` if failed, when calling it from
```

Comment:
Sorry for not pointing it out later, but please note that ChangeLog is not markdown and the \` character has no efect there. I would suggest ` ``mbedtls_aes_crypt_ecb`` ` -> `mbedtls_aes_crypt_ecb()`